app/controllers/post_types_controller.rb

```diff
@@ -10,4 +11,36 @@ def create
 
     redirect_back fallback_location: group_url, alert: post_type.errors.to_a.first
   end
+
+  def index
+    @post_types = @group.post_types.select do |post_type|
```

Comment:
Instead of filtering it here, what do you think about something like this in the `PostType`?
```
COMMON_TYPES = [ECONOMIST_ID, LEADER_POST_ID, PAST_LEADER_ID, DEFAULT_POST_ID, PEK_ADMIN_ID, NEW_MEMBER_ID].freeze
scope :without_common, -> { where.not(id: COMMON_TYPES) }
```
and in the controller
```
@post_types = @group.post_types.without_common
```

Reply:
Yep, it's much cleaner, I'll rewrite it.